hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/PerformanceEvaluation.java

```diff
@@ -1881,15 +1880,15 @@ boolean testRow(final int i, final long startTime) throws IOException {
 
     protected abstract Pair<byte[], byte[]> getStartAndStopRow();
 
-    protected Pair<byte[], byte[]> generateStartAndStopRows(int maxRange) {
-      int start = this.rand.nextInt(Integer.MAX_VALUE) % opts.totalRows;
-      int stop = start + maxRange;
+    protected Pair<byte[], byte[]> generateStartAndStopRows(long maxRange) {
+      long start = ThreadLocalRandom.current().nextLong(Long.MAX_VALUE) % opts.totalRows;
```

Comment:
Here we need to use the Random object so we can reproduce the same sequence when using the same seed.

Reply:
I see, but if I'm not mistaken, we currently don't have any test case that depends on the repeatability of the random sequence. i.e. there is no code in `TestPerformanceEvaluation` that touches `EnvironmentEdgeManager`.

Also, the original comment states that the purpose of the seeding was to ensure that each thread gets a different random sequence, which can be nicely handled by using `ThreadLocalRandom` instead. See https://github.com/apache/hbase/commit/685d6093b0bd8a1e3f6be20aa5dbbf4d4d30d0ab.

So I figured we could just remove the seeding. Do you still think we need to use a Random object that can receive a seed? Please let me know if I'm missing something. Thanks.

Reply:
I used to think we can specify the seed with command line argument...

So here we just want to make sure that different threads have different sequence, use ThreadLocalRandom is OK.